website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,412 @@
+---
+title: 'React Native 0.74 - Yoga Improvements, Preparing for the New Architecture'
+authors: [hurali, alanjhughes, alfonsocj, huntie, cortinico]
+tags: [announcement, release, yoga]
+date: 2024-04-22
+---
+
+Today we're releasing React Native 0.74! This release adds Yoga 3.0, Bridgeless by default under the New Architecture, batched `onLayout` updates, and Yarn 3 as the default package manager for new projects. We are also actioning API removals for the deprecated `PropTypes` and `PushNotificationIOS` APIs, and Android 6.0 is now the minimum supported Android version.
+
+### Highlights
+
+- [Yoga Improvements](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#yoga-improvements)
+- [Preparing for the New Architecture](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#preparing-for-the-new-architecture)
+- [Batched `onLayout` updates (New Architecture)](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#batched-onlayout-updates-new-architecture) (**breaking change**)
+- [Removal of deprecated `PropTypes`](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#removal-of-deprecated-proptypes)
+- [API Changes to PushNotificationIOS (Deprecated)](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#api-changes-to-pushnotificationios-deprecated)
+- [Android SDK minimum bump (Android 6.0)](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#android-sdk-minimum-bump-android-60)
+- [Yarn 3 for New Projects](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#yarn-3-for-new-projects)
+
+### Breaking Changes
+
+- [Breaking Changes](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#breaking-changes)
```

Comment:
[nit] isn't this a bit redundant? 
A title "Breaking Changes" and a bullet list with a single item "breaking changes". What about merging this in the highlights? At the end of the day, breaking changes are something to highlight!

Reply:
It is, however this matches the last 2 announcement posts, with Breaking Changes being a section. Technically these are "Highlighted/Other Breaking Changes".

This time, we don't have any big breaking changes which warrant their own subheading (pending any CLI breaking changes).

Reply:
Resolved by moving some features into Breaking Changes!